images/linux/Ubuntu1804-Readme.md

```diff
@@ -70,11 +66,11 @@
 - Git LFS 3.1.4 (apt source repository: https://packagecloud.io/install/repositories/github/git-lfs)
 - Git-ftp 1.3.1
 - Haveged 1.9.1
-- Heroku 7.60.1
+- Heroku 7.60.2
 - HHVM (HipHop VM) 4.159.0
 - jq 1.5
 - Kind 0.12.0
-- Kubectl 1.23.6
+- Kubectl 4.5.4
```

Comment:
The latest `kubernetes` version is 1.24.0: https://github.com/kubernetes/kubernetes/releases

Reply:
It looks like we output customize version now instead of kubectl
<img width="902" alt="image" src="https://user-images.githubusercontent.com/48208649/166636581-38450aa5-54bc-44a8-ba76-ece1547993cb.png">